packages/x-utils/src/types/__tests__/property-path.types.spec.ts

```diff
@@ -31,6 +31,18 @@ interface Person {
   friends: Person[];
 }
 
+interface AnotherExample {
+  stringKey: string;
+  optionalStringKey?: string;
+  numberKey: number;
+  booleanKey: boolean;
+  objectKey: {
+    innerStringKey: string;
+  };
+}
```

Comment:
Move the interface near the test that uses it. I would do the same with the rest of the interfaces. Unless they are used in every test, it is better to locate them near the one that is using it. Easier to read and easier to maintain.